src/xrGame/trade_factor_parameters_inline.h

```diff
@@ -12,7 +12,7 @@ IC CTradeFactorParameters::CTradeFactorParameters() {}
 IC void CTradeFactorParameters::clear() { m_factors.clear(); }
 IC void CTradeFactorParameters::enable(const shared_str& section, const CTradeFactors& factors)
 {
-    FACTORS::const_iterator I = m_factors.find(section);
+    [[maybe_unused]] auto I = m_factors.find(section);
     VERIFY(I == m_factors.end());
```

Comment:
Currently, this debug check gets into production code and working without doing anything useful on release configurations.
Lets keep it debug only.
```suggestion
    VERIFY(m_factors.find(section) == m_factors.end());
```